### DISCUSSION TITLE
What does the Disabled Protection button actually disable?

### DISCUSSION THREAD

**Discussion by EdinburghWG** [AdguardTeam/AdGuardHome#7078] [Q&A]
As in the title: if you have custom rules, does it disable them, too (i.e. under Filters), or is it just the ones listed under Settings? Or both? Thank you.

**Comment by Lolen10** [comment]
It disables **any** blocking. So no filters, browsing security web service and no parental control web service. It just forwards your requests to the upstream servers without any interference, modification or blocking of your requests.
